src/core/Akka.API.Tests/verify/CoreAPISpec.ApprovePersistence.Net.verified.txt

```diff
@@ -587,7 +587,8 @@ namespace Akka.Persistence
     }
     public sealed class SnapshotMetadata : System.IEquatable<Akka.Persistence.SnapshotMetadata>
     {
-        public static System.DateTime TimestampNotSpecified;
+        [System.ObsoleteAttribute("This constructor is deprecated and will be removed in v1.6. Use the constructor w" +
```

Comment:
A little nitpick, Would be better if we follow english language sentence breaking point here, makes it easier to read.

Reply:
Good point - I can fix that.

Reply:
Ah, no I can't - that's an artifact from the rendering of the API diff. The real code looks like this:

```csharp
[Obsolete("This constructor is deprecated and will be removed in v1.6. Use the constructor with the timestamp parameter instead.", true)]
```